packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -21,13 +21,12 @@ import 'scroll_position.dart';
 import 'scroll_position_with_single_context.dart';
 import 'scrollable.dart';
 
-/// Class that supplies children for the [ListWheelScrollView].
+/// A delegate that supplies children for [ListWheelScrollView].
```

Comment:
I find it quite hard to write clear documentation for these classes.
Since they are pretty similar to SliverChildDelegate, I've used the documentation from there and modified it.
Is it okay to do so?
@DaveShuckerow